examples/server/app/surface3d/surface3d.ts

```diff
@@ -135,12 +136,12 @@ export class Surface3d extends LayoutDOM {
     // types have counterparts, e.g. ``bokeh.core.properties.String`` will be
     // ``p.String`` in the JS implementation. Where the JS type system is not yet
     // as rich, you can use ``p.Any`` as a "wildcard" property type.
-    this.define<Surface3d.Props>(({Unknown, Str, Dict, Ref}) => ({
+    this.define<Surface3d.Props>(({Str, Any, Ref}) => ({
       x:           [ Str ],
       y:           [ Str ],
       z:           [ Str ],
       data_source: [ Ref(ColumnDataSource) ],
-      options:     [ Dict(Unknown), OPTIONS ],
+      options:     [ Any, OPTIONS ],
```

Comment:
You can't remove type information and hope for the best, because this will fail on run time certain situations. Instead, you have to restore this and update the type of `options` in `Surface3d.Props` appropriately, i.e. use `Dict<T>` type from `core/util/types`. Then you have to adjust the use site of `options` to reflect this change (probably `to_object()` from `core/util/object` is the tool for this).